### ISSUE TITLE
Windows PyQt6 QCliboard copy errors out when compiled to exe with Briefcase

### ISSUE THREAD

**Issue by gambhiro** [beeware/briefcase#944]
**Describe the bug**

On Windows, the PyQt6 QClipboard copy (Ctrl+C or .clipboard_setText method) doesn't work when the app is compiled to exe with Briefcase.

It prints this error in the terminal:

```
OleSetClipboard: Failed to set mime data (text/html, text/plain) on clipboard: COM error 0xffffffff800401f0 CO_E_NOTINITIALIZED (CoInitialize has not been called.) (The operation completed successfully.)
```

Same problem when building either with Github Actions, or on a local Windows. I tested on Windows 10 and 11, with briefcase 0.3.11, .10, .9.

The clipboard copy works in the same app when compiled to exe with PyInstaller, and when running the code from source.

The clipboard works in Linux, when compiled to AppImage with Briefcase.

The app fails to start with PyQt 5.15.2 + briefcase 0.3.11, .10, .9, with the error:

```
QWindowsContext: OleInitialize() failed:  "COM error 0xffffffff80010106 RPC_E_CHANGED_MODE (Unknown error 0x080010106)"
[12048:13404:1029/144452.110:FATAL:tsf_text_store.cc(52)] Failed to initialize CategoryMgr.
```

It does work with PyQt5 and briefcase 0.3.8.

PyQt 5.15.2 + briefcase 0.3.8 works, .msi builds, copy works

PyQt 6.4.0 + briefcase 0.3.8 works, .msi builds, copy works

**To Reproduce**

There are two small testing apps here:

- https://github.com/gambhiro/pyqt6-build-example
- https://github.com/gambhiro/pyqt5-build-example

Clone and install the dependencies with `poetry install`.

Build with `briefcase build`, the run the exe (in the terminal to see errors)

The app shows a HTML page in a QWebEnginePage. Select a piece a text, try to copy with either Ctrl+C or Right-click > Copy.

The copy fails (paste will be empty) and the above error about `OleSetClipboard` is shown.

**Expected behavior**

After selecting the text and pressing Ctrl+C, clipboard Ctrl+V should paste the text.

**Comment by freakboy3742** [comment]
Thanks for the report; I wonder if this might be related to #930. We recently introduced a stub binary to Windows apps; the errors you're reporting here seem consistent with the sort of thing you might see on Windows if the app execution mode isn't correct.

**Comment by freakboy3742** [comment]
Closing due a lack of response; I suspect this will have been resolved by the fix we added for #930.
